model/_plugins/notifications/create_config/operations.smithy

```diff
@@ -0,0 +1,22 @@
+//  SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/latest/observing-your-data/notifications/api/"
+)
+
+@xOperationGroup("notifications.create_config")
+@xVersionAdded("2.2")
+@suppress(["HttpUriConflict"])
+@http(method: "POST", uri: "/_plugins/_notifications/configs")
+@documentation("Creates a notification channel configuration.")
```

Comment:
Let's keep it short and simple: `Create channel configuration`, the title of this section in the ref doc works, too.

Reply:
text replaced